icaruscode/CRT/CRTTPCTruthEffMCKalman_module.cc

```diff
@@ -0,0 +1,1721 @@
+	all_side_planes[4].zmin = -407;      all_side_planes[4].zmax = 393;
+	all_side_planes[4].reg_num = 44;     all_side_planes[4].reg_name = "Side East-Center";
+	all_side_planes[4].thin_range = 0;
+	
+	all_side_planes[5].xmin = -534.49;   all_side_planes[5].xmax = -526.22;
+	all_side_planes[5].ymin = -280;      all_side_planes[5].ymax = 450;
+	all_side_planes[5].zmin = 359.34;    all_side_planes[5].zmax = 1159.34;
+	all_side_planes[5].reg_num = 45;     all_side_planes[5].reg_name = "Side East-North";
+	all_side_planes[5].thin_range = 0;
+	
+	all_side_planes[6].xmin = -500;       all_side_planes[6].xmax = 500;
+	all_side_planes[6].ymin = -300;       all_side_planes[6].ymax = 500;
+	all_side_planes[6].zmin = -1134.74;   all_side_planes[6].zmax = -1120.33;
+	all_side_planes[6].reg_num = 46;      all_side_planes[6].reg_name = "Side South";
+	all_side_planes[6].thin_range = 2;
+	
+	all_side_planes[7].xmin = -508.275;   all_side_planes[7].xmax = 508.275;
+	all_side_planes[7].ymin = 80;         all_side_planes[7].ymax = 320;
+	all_side_planes[7].zmin = 1169.72;    all_side_planes[7].zmax = 1177.99;
+	all_side_planes[7].reg_num = 47;      all_side_planes[7].reg_name = "Side North Topmost";
+	all_side_planes[7].thin_range = 2;
+	
+	all_side_planes[8].xmin = -508.275;   all_side_planes[8].xmax = -353.325;
+	all_side_planes[8].ymin = -80;	      all_side_planes[8].ymax = 80;
+	all_side_planes[8].zmin = 1169.72;    all_side_planes[8].zmax = 1177.99;
+	all_side_planes[8].reg_num = 47;      all_side_planes[8].reg_name = "Side North Mid-East";
+	all_side_planes[8].thin_range = 2;
+	
+	all_side_planes[9].xmin = 353.325;    all_side_planes[9].xmax = 508.275;
+	all_side_planes[9].ymin = -80;        all_side_planes[9].ymax = 80;
+	all_side_planes[9].zmin = 1169.72;    all_side_planes[9].zmax = 1177.99;
+	all_side_planes[9].reg_num = 47;      all_side_planes[9].reg_name = "Side North Mid-West";
+	all_side_planes[9].thin_range = 2;
+	
+	all_side_planes[10].xmin = -508.275;   all_side_planes[10].xmax = -380.005;
+	all_side_planes[10].ymin = -160;       all_side_planes[10].ymax = -80;
+	all_side_planes[10].zmin = 1169.72;    all_side_planes[10].zmax = 1177.99;
+	all_side_planes[10].reg_num = 47;      all_side_planes[10].reg_name = "Side North Bottom-East";
+	all_side_planes[10].thin_range = 2;
+	
+	all_side_planes[11].xmin = 380.005;    all_side_planes[11].xmax = 508.275;
+	all_side_planes[11].ymin = -160;       all_side_planes[11].ymax = -80;
+	all_side_planes[11].zmin = 1169.72;    all_side_planes[11].zmax = 1177.99;
+	all_side_planes[11].reg_num = 47;      all_side_planes[11].reg_name = "Side North Bottom-West";
+	all_side_planes[11].thin_range = 2;
+*/
+
+
+}//end definition of void icarus::CRTTPCTruthEffMCKalman::findIntersectingCRTVolume(recob::Track trk, TVector3 &retvec, TVector3 &endpoint, int &likely_CRT_region)
+
+void icarus::CRTTPCTruthEffMCKalman::endpoint_PCA_ana(recob::Track trk, bool usestartpt, TVector3 &bestfit_dir, TVector3 &bestfit_pos){
```

Comment:
It seems like the purpose of this function is to update the PCA best-fit position and direction. If so, and if it's not too much trouble, I'd suggest naming the function something like "GetPCAParams" or something.